megazords/ios-rust/MozillaTestServices/glean.udl

```diff
@@ -0,0 +1,549 @@
+namespace glean {
```

Comment:
Curious why we needed to add a second `udl` here - is this the same `udl` in glean? Is it possible to point to the submodule in all of our scripts instead of duplicating the UDL?

Reply:
Yes, that was my intention, Glean now has two udl files and does some swapping between them when building the iOS megazord (via somewhat hacky use of file renaming, but it works for now). I think I just forgot to remove this one once I moved the udl needed to build the iOS megazord compatible Glean into the Glean repo.